ablator/analysis/results.py

```diff
@@ -119,13 +255,31 @@ def _assert_cat_attributes(self, categorical_attributes: list[str]):
 
     @property
     def metric_names(self) -> list[str]:
+        """
+        get the list of all optimize metric names
```

Comment:
I think `self.metric_map.values()' returns optimization direction, not metric names